src/firefly/js/drawingLayers/SelectedShape.js

```diff
@@ -2,4 +2,11 @@
  * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
  */
 import Enum from 'enum';
+
+/**
+ * @typedef SelectedShape
+ * enum can be rect, circle
+ * @prop rect
+ * @prop circle
+ */
```

Comment:
Can't find this in /docs/js/.  I don't think it got generated.  It would be interesting to see how the generated docs look.  Because as is, it looks like an ordinary untyped typedef with untyped props.